appsettings.json

```diff
@@ -5,5 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
-}
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "LibraryConnection": "Server=127.0.0.1;Database=BookDb;User=root;Password=idongesit@1998;"
+  }
```

Comment:
This should be moved to the appsettings.Development.json file, as it's a sensitive variable that varies according to the environment in which it's deployed.